### DISCUSSION TITLE
React native support

### DISCUSSION THREAD

**Discussion by vsamerv** [microsoft/FluidFramework#15096] [Q&A]
Hello, I am trying to utilize Fluid Framework between Web App (Angular) and Mobile App (React Native). Is it even possible to use Fluid Framework with React Native? Thank you.

**Comment by hvpatil** [comment]
Hi @vsamerv,
Please follow https://fluidframework.com/docs/recipes/react/ for guidance to use Fluid Framework with React.  It has example to connect Fluid Data layer with view layer made in React.

**Reply by vsamerv** [reply]
hi @hvpatil,

Thanks for the link but I am looking for React Native (for mobile) guidance. Is it available yet?

**Reply by hvpatil** [reply]
Fluid Framework doesn't have any dependency on view layer. Whether client is using react DOM or react native to render the UI element is client's choice. Do you have any specific concerns while working with react native while connecting Fluid Data layer?

**Reply by vsamerv** [reply]
I am still in R&A phase and could not find any “official” confirmation and documentation for React Native support. That’s it. 
Thanks for the details.

**Reply by hvpatil** [reply]
It should work. We don't have any testing done on react-native yet.
